### PR TITLE
Added Python model REST protocol test on triton for Kserve

### PR DESCRIPTION
**Pull request by Raghul-M**

### CODE REVIEW

**Comment by Raghul-M** [comment]
Verified with Jenkins Build 1302

File: ods_ci/tests/Tests/1000__model_serving/1009__model_serving_triton_on_kserve/1009__model_serving_triton_on_kserve.robot [hunk 1]
```diff
@@ -61,8 +63,8 @@ Test Onnx Model Rest Inference Via UI (Triton on Kserve)    # robocop: off=too-l
     Create Data Science Project    title=${PRJ_TITLE}    description=${PRJ_DESCRIPTION}
     ...    existing_project=${FALSE}
     Open Dashboard Settings    settings_page=Serving runtimes
-    Upload Serving Runtime Template    runtime_filepath=${ONNX_RUNTIME_FILEPATH}
-    ...    serving_platform=single      runtime_protocol=REST
+    #Upload Serving Runtime Template    runtime_filepath=${ONNX_RUNTIME_FILEPATH}
+    #...    serving_platform=single      runtime_protocol=REST
```
Comment (tarukumar) [inline ods_ci/tests/Tests/1000__model_serving/1009__model_serving_triton_on_kserve/1009__model_serving_triton_on_kserve.robot:67]: why above line is commented ?

File: ods_ci/tests/Tests/1000__model_serving/1009__model_serving_triton_on_kserve/1009__model_serving_triton_on_kserve.robot [hunk 2]
```diff
@@ -240,6 +242,36 @@ Test KERAS Model Inference Via UI(Triton on Kserve)
     ...  AND
     ...  Delete Serving Runtime Template From CLI    displayed_name=triton-keras-rest
 
+Test Python Model Rest Inference Via UI (Triton on Kserve)    # robocop: off=too-long-test-case
+    [Documentation]    Test the deployment of an python model in Kserve using Triton
+    [Tags]    Sanity    RHOAIENG-15374      RunThisTest
```
Comment (tarukumar) [inline ods_ci/tests/Tests/1000__model_serving/1009__model_serving_triton_on_kserve/1009__model_serving_triton_on_kserve.robot:247]: Why `RunThisTest` tag. This is not right

Comment (Raghul-M) [reply ods_ci/tests/Tests/1000__model_serving/1009__model_serving_triton_on_kserve/1009__model_serving_triton_on_kserve.robot:247]: will remove it now

File: ods_ci/tests/Tests/1000__model_serving/1009__model_serving_triton_on_kserve/1009__model_serving_triton_on_kserve.robot (hunk 1 above)
Comment (Raghul-M) [reply ods_ci/tests/Tests/1000__model_serving/1009__model_serving_triton_on_kserve/1009__model_serving_triton_on_kserve.robot:67]: will uncomment the lines now

**Comment by bdattoma** [comment]
@Raghul-M @tarukumar given that the number of UI tests for serving runtime seems to be increasing, and each test can take a relatively high time to run (15 min approx? correct me if I'm wrong), should we consider moving them to Tier2? Leaving maybe one in Sanity. The tests share the same user path at the end of the day, so for Sanity one should be sufficient. Wdyt?

**Comment by Raghul-M** [comment]
> @Raghul-M @tarukumar given that the number of UI tests for serving runtime seems to be increasing, and each test can take a relatively high time to run (15 min approx? correct me if I'm wrong), should we consider moving them to Tier2? Leaving maybe one in Sanity. The tests share the same user path at the end of the day, so for Sanity one should be sufficient. Wdyt?

@bdattoma  I agree, Berto. We still have several UI-related model-serving tests, so it's a good idea to move some of them to Tier 2. Many of these test cases are similar but involve different models. We could consider moving a few test cases from each scenario to the sanity suite to maintain coverage .

**Comment by bdattoma** [comment]
> > @Raghul-M @tarukumar given that the number of UI tests for serving runtime seems to be increasing, and each test can take a relatively high time to run (15 min approx? correct me if I'm wrong), should we consider moving them to Tier2? Leaving maybe one in Sanity. The tests share the same user path at the end of the day, so for Sanity one should be sufficient. Wdyt?
> 
> @bdattoma I agree, Berto. We still have several UI-related model-serving tests, so it's a good idea to move some of them to Tier 2. Many of these test cases are similar but involve different models. We could consider moving a few test cases from each scenario to the sanity suite to maintain coverage .

ok thanks. Are you going to move the test in this PR to tier2 or is it going to remain in Sanity?

**Comment by Raghul-M** [comment]
> > > @Raghul-M @tarukumar given that the number of UI tests for serving runtime seems to be increasing, and each test can take a relatively high time to run (15 min approx? correct me if I'm wrong), should we consider moving them to Tier2? Leaving maybe one in Sanity. The tests share the same user path at the end of the day, so for Sanity one should be sufficient. Wdyt?
> > 
> > 
> > @bdattoma I agree, Berto. We still have several UI-related model-serving tests, so it's a good idea to move some of them to Tier 2. Many of these test cases are similar but involve different models. We could consider moving a few test cases from each scenario to the sanity suite to maintain coverage .
> 
> ok thanks. Are you going to move the test in this PR to tier2 or is it going to remain in Sanity?

I will move this one  from Sanity to Tier 2

File: ods_ci/tests/Resources/Page/Components/Menu.robot [hunk 3]
```diff
@@ -10,6 +10,7 @@ Navigate To Page
    ...    ${timeout}=10s
    Wait Until Element Is Visible    //div[@id="page-sidebar"]    timeout=${timeout}
    Wait Until Page Contains    ${menu}
+   ${menu}=    Set Variable If    "${menu}" == "Deployed models"    Model Serving    ${menu}
```
[the remark this answers is not in the document]
Comment (bdattoma) [reply ods_ci/tests/Resources/Page/Components/Menu.robot:13]: hm if the text has changed, then the other tests should be changed as well. Have you checked how many test cases would be impacted?

**Comment by Raghul-M** [comment]
Verified with Jenins Build 1871

**Review by bdattoma — APPROVED** [review]

**Review by tarukumar — APPROVED** [review]

**Pull request merged**